package.json

```diff
@@ -69,6 +69,7 @@
     "postcss-flexbugs-fixes": "^3.3.0",
     "postcss-loader": "2.1.3",
     "promise": "^8.0.1",
+    "pug": "^2.0.3",
```

Comment:
Is this package needed?

Reply:
@bbashy https://github.com/jfurrow/flood/blob/4346c0d6b57fe503d21348246fae20a217d65165/server/app.js#L25